packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec

```diff
@@ -9,7 +9,7 @@
 %global release 2
```

Comment:
Needs to be set back to 1

Reply:
Updated 👍🏼 btw what does `release 1/2` mean?

Reply:
Don't forget the changelog entry. That must match.

The release is a version number for the packaging. It allows packagers to make incremental changes on the same original release. https://github.com/theforeman/foreman-packaging/commit/3e4c0f1924a8760df55b68a0fe826f48f34c3b32 is an example of that.

Reply:
Thanks for the explanation, done.